tests/tests.py

```diff
@@ -11,7 +11,7 @@
 class DNSTest(unittest.TestCase):
 
     def setUp(self):
-        self.channel = pycares.Channel(timeout=1.0, tries=1)
+        self.channel = pycares.Channel(timeout=5.0, tries=1)
```

Comment:
I understand why you want to test this on a highly available server. However, since we're testing the actual contents of the TXT records in these tests (as opposed to tests that mainly just check the return type), we will very likely face the situation were the records will change at some point.
Finding a test for chunked TXT records on a server like Google, will not really help in that regard, as they are susceptible to change as well. Not sure which road to go with these tests. Any thoughts?